schemas-generated/flatbuffers/opendatafabric.fbs

```diff
@@ -323,6 +323,22 @@ table EnvVar {
   value: string;
 }
 
+////////////////////////////////////////////////////////////////////////////////
+// MqttTopicSubscription
+// https://github.com/kamu-data/open-data-fabric/blob/master/open-data-fabric.md#mqtttopicsubscription-schema
+////////////////////////////////////////////////////////////////////////////////
+
+enum MqttQos: int32 {
+  AtMostOnce,
+  AtLeastOnce,
+  ExactlyOnce,
+}
+
+table MqttTopicSubscription {
+  path: string;
+  qos: MqttQos = null;
```

Comment:
**Q:** could you please tell me where to learn more about QOS in terms of MQTT?

Reply:
I read about them here:
https://www.hivemq.com/blog/mqtt-essentials-part-6-mqtt-quality-of-service-levels/